src/main/java/com/fasterxml/jackson/databind/jsontype/BasicPolymorphicTypeValidator.java

```diff
@@ -239,14 +247,75 @@ public boolean match(String clazzName) {
             });
         }
 
-        public BasicPolymorphicTypeValidator build() {
+        /**
+         * Validates and Constructs the BasicPolymorphicTypeValidator.
+         * Unlike build() no check is done to ensure the Matchers added to this validator will allow Object.class
+         * or Serializable.class and their subtypes to be deserialised.
+         * As the name suggests, this is an insecure operation. Please think very carefully before calling this method
+         * rather than build(), especially if the caller is outside the trust boundary of this application. As doing
+         * so will leave your application open to vulnerabilities including remote code execution.
+         * @return the BasicPolymorphicTypeValidator.
+         */
+        public BasicPolymorphicTypeValidator build_insecure()  {
```

Comment:
`build_insecure()` is not following any naming convention, but in general I'll add my notes on why I don't think this is the way to go.